### ISSUE TITLE
"Guild Check Auth Error" when missing radia settings

### ISSUE THREAD

**Issue by inkfarer** [inkfarer/ipl-overlay-controls#44]
Full log: 

`error: [ipl-overlay-controls] Guild Check Auth Error: TypeError [ERR_HTTP_INVALID_HEADER_VALUE]: Invalid value "undefined" for header "Authorization"`

Occurs when NodeCG is started and `radia` is unset in the bundle configuration. Found on a fresh install, so `radiaSettings` has only default values.

**Comment by vlee489** [comment]
Error
```ps
PS B:\IPL\ocp3.0> nodecg start
[nodecg] No config found, using defaults.
info: [nodecg/lib/server] Starting NodeCG 1.8.1 (Running on Node.js v14.15.4)
warn: [ipl-overlay-controls] Bundle low-ink-overlays expects version ^3.0.0 of ipl-overlay-controls! The installed version is 2.7.0.
info: [nodecg/lib/server/extensions] Mounted ipl-overlay-controls extension
info: [nodecg/lib/server] NodeCG running on http://localhost:9090
error: [ipl-overlay-controls] Guild Check Auth Error: TypeError [ERR_HTTP_INVALID_HEADER_VALUE]: Invalid value "undefined" for header "Authorization"
error: [ipl-overlay-controls] Guild Check Auth Error: TypeError [ERR_HTTP_INVALID_HEADER_VALUE]: Invalid value "undefined" for header "Authorization"
```

**Comment by inkfarer** [comment]
Resolved in 3.0 (Not sure exactly when...)
